packages/commonwealth/server/migrations/20240815110642-contests-abi-fix.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+const { QueryTypes } = require('sequelize');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
```

Comment:
if you are only using sql scripts, you can try a single query with all the statements to keep it shorter

Reply:
I'm leaning towards simplicity rather than succinctness since testing migrations is generally a bit painful